environment.yml

```diff
@@ -14,6 +14,8 @@ dependencies:
 
   # building
   - cython>=0.28.2
+  - cxx-compiler
+  - zlib
```

Comment:
can you add a short description why those are needed

Reply:
Sure! I will add the explanation, actually the ``zlib`` we can remove since it is already a dependency of other packages

Reply:
I also think that the ``conda-forge`` channel should be prioritized by the defaults
since not all of the packages are present on the default channel and the project is using the missing packages from ``conda-forge``, you might see some problems in the future (unlikely) but it is better to avoid any kind of problem

But I can create another PR for that